packages/fluentui/accessibility/src/behaviors/SplitButton/splitButtonBehavior.ts

```diff
@@ -56,4 +56,8 @@ const splitButtonBehavior: Accessibility = props => {
   };
 };
 
+export type SplitButtonBehaviorProps = {
```

Comment:
This is too simple to be truth 😸 

1. This behavior inherits `menuButtonBehavior` which inherits `popupBehavior`:
```suggestion
export type SplitButtonBehaviorProps = Pick<MenuButtonBehaviorProps, 'open' | OTHER_REQUIRED_PROPS>
```

I suggest to reach @silviuaavram to ask about really required props for this behavior and them them to `mapPropsToBehavior`.
2. Use it in `Accessibility` generic: `Accessibility<SplitButtonBehaviorProps>`

Reply:
I am already using `accessibility?: Accessibility<SplitButtonBehaviorProps>;`

Reply:
Talked with @silviuaavram and `open` is enough. So as agreed I changed to 
`export type SplitButtonBehaviorProps = Pick<MenuButtonBehaviorProps, 'open'>;`